1.1/MetadataGranularities/README.md

```diff
@@ -0,0 +1,200 @@
+
+## Screenshot
+
+![MetadataGranularities](screenshot/MetadataGranularities.png)
+
+## Sandcastle Code
+
+```JavaScript
+var viewer = new Cesium.Viewer('cesiumContainer');
+
+// Create the tileset, and set its model matrix to move it
+// to a certain position on the globe
+var tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url : 'http://localhost:8003/1.1/MetadataGranularities/tileset.json',
+    maximumScreenSpaceError: 1,
+    debugShowBoundingVolume: true
+}));
+tileset.modelMatrix = Cesium.Transforms.eastNorthUpToFixedFrame(
+  Cesium.Cartesian3.fromDegrees(-75.152325, 39.94704, 0.0));
+
+// Create the label that will display metadata information
+var labelEntity = viewer.entities.add({
+  label: {
+    showBackground: true,
+    font: "14px monospace",
+    horizontalOrigin: Cesium.HorizontalOrigin.LEFT,
+    verticalOrigin: Cesium.VerticalOrigin.TOP,
+    pixelOffset: new Cesium.Cartesian2(15, 0),
+    // Workaround to always render label on top:
+    disableDepthTestDistance: Number.POSITIVE_INFINITY
+  }
+});
+
+// Adds metadata information to the label, with 
+// the given title indicating which granularity
+// of metadata this is. The given metadata object
+// can be a TilesetMetadata, TileMetadata, 
+// GroupMetadata, or ContentMetadata.
+var addMetadataToLabel = function(title, metadata) {
+  if (!Cesium.defined(metadata)) {
+    labelEntity.label.text += "(No " + title + ")\n";
+    return;
+  }
+  var propertyKeys = metadata.getPropertyIds();
+  if (!Cesium.defined(propertyKeys)) {
+    labelEntity.label.text += "(No properties for " + title + ")\n";
+  }
+  labelEntity.label.text += title + ":\n";
+  for (var i=0; i<propertyKeys.length; i++) {
+    var propertyKey = propertyKeys[i];
+    var propertyValue = metadata.getProperty(propertyKey);          
+    //console.log("propertyKey", propertyKey);
+    //console.log("propertyValue", propertyValue);
+    labelEntity.label.text += "  " + propertyKey + ": "+ propertyValue;
+    labelEntity.label.text += "\n";
+  }
+};
+
+// (NOTE: The way how metadata is obtained may be refactored,
+// see https://github.com/CesiumGS/cesium/issues/10015 )
+
+// Given an object that was obtained via Scene#pick, examine it 
+// to see whether it contains tile metadata.
+// If the object contains TileMetadata, then this metadata 
+// is returned. 
+// Otherwise, 'undefined' is returned.
+var obtainTileMetadata = function(picked) {
+  if (!Cesium.defined(picked)) {
+    return undefined;
+  }
+  if (!Cesium.defined(picked.content)) {
+    return undefined;
+  }
+  if (!Cesium.defined(picked.content.tile)) {
+    return undefined;
+  }
+  if (!Cesium.defined(picked.content.tile.metadata)) {
+    return undefined;
+  }
```

Comment:
You might be able to reduce the amount of code with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

And speaking of using newer JS features, use let/const instead of var and run through prettier to fix any formatting inconsistencies.

Reply:
I think optional chaining is not supported... in sandcastles...? But will use `const`, that seems to work...

Reply:
https://github.com/CesiumGS/cesium/pull/10220 should fix the warning, just had to update some configs in sandcastle.

Also there's still one `var` in the code that should be a `let`.